scriptmodules/supplementary/runcommand.sh

```diff
@@ -19,7 +19,7 @@ function _update_hook_runcommand() {
 }
 
 function depends_runcommand() {
-    local depends=()
+    local depends=(python-pyudev)
```

Comment:
I don't have a problem if it did it without using the code - might be worth looking how the module does it since your only use a single function of it anyway and then access sysfs manually - we handle ini manually, and the actual joystick handling is low level rather than via pygame etc.